test_conformance/CMakeLists.txt

```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
```

Comment:
I'm guessing you didn't intend to keep this.

Reply:
Yeah it was mistake,I'll remove that